### PR TITLE
move multi-network to GA

### PR DESCRIPTION
**Pull request by kkuchima**
This PR is to add terraform support for Multi-network GA on GKE cluster.

The Multi-network in Terraform beta was added to the magic module in the PR #8385.  
This PR basically moves the `enable_multi_networking` field in `google_container_cluster` and `additional_node_network_configs` `additional_pod_network_configs` fields in `google_container_node_pool` to GA.  

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
container: promoted `enable_multi_networking` to GA in the `google_container_cluster` resource
```

```release-note:enhancement
container: promoted `additional_node_network_configs` and `additional_pod_network_configs` fields to GA in the `google_container_node_pool` resource
```

### CODE REVIEW

**Review by NickElliot — CHANGES_REQUESTED** [review]
could you resolve the conflicts? thanks!

**Comment by kkuchima** [comment]
@NickElliot Thanks, resolved the conflicts.

**Review by NickElliot — CHANGES_REQUESTED** [review]
The error is an unexpected `<% end %>` tag, make sure you remove all the corresponding ends for each removed `unless` tag!

File: mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb [hunk 1]
```diff
@@ -699,7 +700,6 @@ resource "google_container_cluster" "cluster" {
 }
 `, clusterName, clusterName)
 }
-<% end -%>
```
Comment (marblejenka) [inline mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb:702]: Should this line be deleted? The following log suggests that the `end` matching is not correct.

```
2024-07-03 05:20:27.234 JST
Step #11 - "tpgb-head": I, [2024-07-02T20:20:18.269515 #170] INFO -- : Compiling common files for terraform
2024-07-03 05:20:27.234 JST
Step #11 - "tpgb-head": #<Thread:0x00007f01c81a7590 /mm-terraform-beta-head/mmv1/provider/terraform.rb:189 run> terminated with exception (report_on_exception is true):
2024-07-03 05:20:27.234 JST
Step #11 - "tpgb-head": /usr/local/lib/ruby/3.1.0/erb.rb:905:in `eval': (erb):11083: syntax error, unexpected end-of-input, expecting `end' (SyntaxError)

**Review by NickElliot — APPROVED** [review]
LGTM!

**Comment by kkuchima** [comment]
@NickElliot @marblejenka Thanks for your help!

**Pull request merged**